src/CmsCacheBundle/Resources/config/services.xml

```diff
@@ -5,9 +5,9 @@
 
     <services>
 
-        <service id="chameleon_system_cms_cache.cache_database_connection" alias="database_connection" />
+        <service id="chameleon_system_cms_cache.cache_database_connection" alias="database_connection"  public="true"/>
 
-        <service id="chameleon_system_cms_cache.cache" class="esono\pkgCmsCache\Cache">
+        <service id="chameleon_system_cms_cache.cache" class="esono\pkgCmsCache\Cache" public="true">
```

Comment:
It's because services will be private by default in Symfony 4 and it will no longer be possible to retrieve private services from the service container (which affects our ServiceLocator).